File: 1000/1840C/main.java
import java.util.*;
public class main
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        
        while(t-- > 0)
        {
            int n = sc.nextInt();
            int k = sc.nextInt();
            int q = sc.nextInt();
            
            long ans = 0;
            int cnt = 0;
            
            for(int i=0;i<n;i++)
            {
                int num = sc.nextInt();
                if(num <= q)
                    cnt++;
                else
                    cnt = 0;
                
                if(cnt >= k)
                    ans = ans + (cnt-k+1);
            }
            
            System.out.println(ans);
        }
    }
}

File: 1000/1840C/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int t;
    cin >> t;

    while (t--) 
    {
        int n, k, q;
        cin >> n >> k >> q;

        long long ans = 0;
        int cnt = 0;

        for (int i = 0; i < n; i++) 
        {
            int num;
            cin >> num;

            if (num <= q)
                cnt++;
            else
                cnt = 0;

            if (cnt >= k) 
                ans += (cnt - k + 1);
        }

        cout << ans << "\n";
    }

    return 0;
}
